src/style/app.module.css

```
@@ -616,54 +588,18 @@
-  color: var(--brown-color);
-  border-color: var(--grey-bg-color);
-  --bs-btn-active-bg: var(--dark-blue);
-  --bs-btn-active-border-color: var(--grey-bg-color);
-}
-
-.editButton:is(:hover, :active) {
-  background-color: var(--dark-blue);
-  border-color: var(--search-button-border);
+.profileImage {
+  display: flex;
+  justifycontent: center;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)